Remant/Deep - Conversation Option and Remnant Awareness (#5111)
endless-sky/endless-sky: data/human/deep missions.txt

	source "Valhalla"
	waypoint "Cardea"
	waypoint "Insitor"
	waypoint "Segesta"
	waypoint "Aescolanus"
	cargo "astronomical recorders" 5
	blocked `You've returned to <planet>, but Ivan informs you that you'll need <capacity> available for this next mission. Return when you have the available space.`
	to offer
		or
			has "Deep: Remnant A: done"
			and
				has "Deep: Remnant: Keystone Research: done"
				not "Deep: Remnant A: offered"
				not "Deep: Remnant 1A: offered"
				not "Deep: Remnant 1B: offered"
				not "Deep: Remnant 1C: offered"
				or
					has "deep: knew of stones before scientists"
					has "First Contact: Remnant: offered"
	on offer
		conversation "deep: remnant"
	on visit
		dialog `You've returned to <planet>, but you haven't traveled through all the wormholes yet. Return to the strange region of space and travel through the loop of wormholes.`

conversation "deep: remnant"
	branch "on second mission" "on first mission"
		has "Deep: Remnant A: done"
	
	label "on first mission"

> 	`You contact Ivan after you land and he sends you directions to his research lab, located on the outskirts of the city. When you arrive you hardly have time to greet Ivan, Rayna, and the others before Ivan begins discussing what they have learned from the Quantum Keystones.`

Reviewer: ```suggestion
	`You contact Ivan after you land and he sends you directions to his research lab, located on the outskirts of the city. When you arrive, you hardly have time to greet Ivan, Rayna, and the others before Ivan begins discussing what they have learned from the Quantum Keystones.`
```

	branch "knew before" "didn't know before"
		has "deep: knew of stones before scientists"
	
	label "knew before"
	`	"As you mentioned before, a single Keystone is sufficient to enter the wormhole," Ivan says. "We've determined that there is a stabilization effect, either on the ship or the wormhole, that allows matter to cross the threshold. However, we'll need much more time and data to fully understand the true mechanism.`
	branch remnant1
		has "deep: did reveal remnant"
	`	"Last time, you mentioned that there were alien creatures able to survive the vacumm of space, correct?"`
	`	You nod.`
	`	"Right. Well, it would be extremely interesting if we were to learn more about these alien creatures.`
		goto end

	label remnant1
	`	"Last time, you mentioned that there were humans living in this space, the... the 'Remainder?'"`
	`	"Remnant," you say.`

> 	`	"Right, that was it. Well, it would be extremely interesting if we were to learn more about this people.`

Reviewer 2: Shouldn't this read ```"Right, that was it. Well, it would be extremely interesting if we were to learn more about these people.```

Reviewer 3: When "people" refers to a society/collective group rather than a bunch of individuals, I think it can be treated as singular instead of plural.

Reviewer 2: In conversation though?  Also, missing a double quote at the end (in case that wasn't obvious).

Reviewer 3: Don't need a quotation mark; the speaker continues in the next line after this (`label end`). It's a small thing, but it's rule of thumb for multiple paragraphs with one speaker to have no ending quotation mark until the quotation is entirely completed. As far as whether "people" is right, I agree it does sound strange if you're not used to hearing it (it's not very common), but I don't think I've ever heard that it's wrong in American English. Could you link something that backs that point up?

Reviewer 2: Apologies, I didn't realize the conversation continued.  But then shouldn't the starting quote in the end conversation not be there or is that to clarify continuing speaking.

Sadly I cannot.  Everything I've found states using the singular "this" is correct.  Many ignore this for the same reason I do but it is not official in any way.

Reviewer 3: Your second take is right; [here's an article on it](http://baxtercommunications.nl/quotation-marks-multi-paragraph-rule/#.X5XyJFMpAwA).
The singular "people" is a strange case. English is strange in general, I guess.

Author: > English is strange in general, I guess.

Yeah, English is definitely strange. Even worse when we've got multiple contradictory variants of it that don't *sound* contradictory, but definitely *look* it.

	branch friendly
		has "event: ember waste label"
	apply
		set "deep: did reveal remnant"
	`	You then tell them about the strange-looking humans who communicate in song and sign language that attacked you after you wouldn't submit to a blood test.`
		goto remnant2
				
	label friendly
	choice
		`	(Reveal the existence of the Remnant.)`
		`	(Don't reveal the existence of the Remnant.)`
			goto secret
	apply
		set "deep: did reveal remnant"
	`	You then tell them about the Remnant, a group of friendly humans who fled human space during the Alpha Wars and have advanced independently of humanity for centuries, and how they call the region of space the "Ember Waste."`
		goto remnant2

	label secret
	apply
		set "deep: did not reveal remnant"

> 	`	The group of scientists looks captivated the entire time, almost like kindergarten children listening to a fairy tale. When you finish talking, they immediately chatter among themselves, seeming to be most interested in the alien creaturess which inhabit the region. They seem to agree that they need to learn more about these aliens.`

Reviewer: ```suggestion
	`	The group of scientists looks captivated the entire time, almost like kindergarten children listening to a fairy tale. When you finish talking, they immediately chatter among themselves, seeming to be most interested in the alien creatures which inhabit the region. They seem to agree that they need to learn more about these aliens.`
```

	`	"We can seek to learn more about these alien creatures later," Ivan says.`
		goto end
		
	label remnant2
	`	The group of scientists looks captivated the entire time, almost like kindergarten children listening to a fairy tale. When you finish talking, they immediately chatter among themselves, seeming to be most interested in the human beings which inhabit the region. They seem to agree that they need to learn more about these humans, and more specifically about the technology which they have developed.`
	`	"We can seek to learn more about this group of humans later," Ivan says.`

	label end

> 	`	"But, first and foremost, we should continue our work on understanding these wormholes. You mentioned a strange series of wormholes. Their behavior is definitely noteworthy. Could you return to this region of space while carrying a suite of astronomical instruments? If you travel through these wormholes, we may learn more about what makes them tick."`

Reviewer: ```suggestion
	`	"But, first and foremost, we should continue our work on understanding these wormholes. You mentioned a strange series of wormholes. Their behavior is definitely noteworthy. Could you return to this region of space while carrying a suite of astronomic instruments? If you travel through these wormholes, we may learn more about what makes them tick."`
```Subjective change for clarity because astronomical also means big.

	`	"We can seek to learn more about these alien creatures later," Ivan says.`
		goto end
		
	label remnant2
	`	The group of scientists looks captivated the entire time, almost like kindergarten children listening to a fairy tale. When you finish talking, they immediately chatter among themselves, seeming to be most interested in the human beings which inhabit the region. They seem to agree that they need to learn more about these humans, and more specifically about the technology which they have developed.`
	`	"We can seek to learn more about this group of humans later," Ivan says.`

	label end
	`	"But, first and foremost, we should continue our work on understanding these wormholes. You mentioned a strange series of wormholes. Their behavior is definitely noteworthy. Could you return to this region of space while carrying a suite of astronomic instruments? If you travel through these wormholes, we may learn more about what makes them tick."`

> ```Subjective change for clarity because astronomical also means big.

Reviewer 4: ```suggestion
```

> # Branch D occurs when the player knows about the Remnant before the start of the mission and decided to not reveal it to the Deep researchers.

Reviewer 5: ```suggestion
# Branch D occurs when the player knew about the Remnant before the start of the mission and decided to not reveal it to the Deep researchers.
```
Tense agreement.

	landing
	name `Analyze Series of Wormholes`
	description `Scan the series of wormholes located in the Ember Waste`
	source "Valhalla"
	waypoint "Cardea"
	waypoint "Insitor"
	waypoint "Segesta"
	waypoint "Aescolanus"
	cargo "astronomical recorders" 5
	to offer
		has "Deep: Remnant: Keystone Research: done"
		has "deep: knew of stones before scientists"
		has "First Contact: Remnant: offered"
		not "Deep: Remnant 1A: offered"
		has "deep: did not reveal remnant"
	on offer
		conversation
			`You contact Ivan after you land and he sends you directions to his research lab, located on the outskirts of the city. When you arrive you hardly have time to greet Ivan, Rayna, and the others before Ivan begins discussing what they have learned from the Quantum Keystones.`
			`	"As you mentioned before, a single Keystone is sufficient to enter the wormhole. We've determined that there is a stabilization effect, either on the ship or the wormhole, that allows matter to cross the threshold. However, we'll need much more time to fully understand the true mechanism." Ivan asks, "Last time, you mentioned that there was an odd series of wormholes?"`
			`	"Yes. They kind of make a train of sorts."`

> 			`	"Right, definitely noteworthy. Well, we've decided that we need to know more about this these wormholes, as they could be significant. Could you make the circuit around this loop while carrying a suite of astronomical instruments? "`

Reviewer 5: ```suggestion
			`	"Right, definitely noteworthy. Well, we've decided that we need to know more about these wormholes, as they could be significant. Could you make the circuit around this loop while carrying a suite of astronomical instruments? "`
```

			choice
				`	"Sure, I'm willing to do it."`
					accept
				`	"Sorry, you'll need to find someone else."`
					decline
	on visit
		dialog `You have returned to <planet>, but you have not yet visited each system in the wormhole series. Go back and complete the set.`
	on complete
		event "deep: scan log research" 14

> 		log `Scanned a series of wormholes beyond the wormhole in Terminus and handed the scan logs over to Ivan and his team.`

Reviewer 5: ```suggestion
		log `Scanned a train of wormholes in the Ember Waste and handed the scan logs over to Ivan and his team.`
```
it seemed awkward to talk about series of wormholes beyond the wormhole. Also, if we're going to make <wormhole train> an official term, reinforcing it in a few more places seems like a good idea.

	on complete
		event "deep: scan log research" 14
		log `Scanned Remnant ships beyond the wormhole in Terminus and handed the scan logs over to Ivan and his team.`
		dialog `The team is visibly excited when you return with the scanner logs. "Give us a few weeks to look these over," Ivan says. "Then we will contact you about what we want you to do next."`

conversation "deep: remnant c"
	`The team of scientists is already waiting in the spaceport when you arrive. They remove the equipment from your ship and immediately being analyzing the results after returning to the lab.`

> 	`	"At first glance this series of wormholes seems no different than the one that is used to enter this region of space, but it will require some time for us to go over all the data the sensors collected," Ivan explains to you.`

Reviewer: ```suggestion
	`	"At first glance, this series of wormholes seems no different than the one that is used to enter this region of space, but it will require some time for us to go over all the data the sensors collected," Ivan explains to you.`
```

	branch revealed
		has "deep: did reveal remnant"

> 	`	"While you wait, though, I have another job you could do. You mentioned something about strange alien creatures beyond the wormhole that seemed to be able to survive in the vacuum of space. Would you be able to return there and scan them for us? These alien creatures won't have 'outifts' for the scanners to pick up, but a typical outfits scanner is likely to give us the most information."`

Reviewer: ```suggestion
	`	"While you wait, though, I have another job you could do. You mentioned something about strange alien creatures beyond the wormhole that seemed to be able to survive in the vacuum of space. Would you be able to return there and scan them for us? These alien creatures won't have 'outfits' for the scanners to pick up, but a typical outfits scanner is likely to give us the most information."`
```

		goto end

	label revealed

> 	`	"While you wait, though, I have another job you could do. You mentioned something about a group of humans, or at least human-like beings, beyond the wormhole. Would you be able to return there and scan their ships for us? I'd be interested in seeing what an outfits scanner returns. You need not contact them, simply find a couple ships and scan them so that we might learn more about their capabilities."`

Reviewer: ```suggestion
	`	"While you wait, though, I have another job you could do. You mentioned something about a group of humans, or at least human-like beings, beyond the wormhole. Would you be able to return there and scan their ships for us? I'd be interested in seeing what an outfits scanner returns. You need not contact them, simply find a couple of ships and scan them so that we might learn more about their capabilities."`
```

	label end
	choice

> 		`	"Sounds like a good plan. I'll be back as fast as I can."`

Reviewer: ```suggestion
		`	"Sounds like a good plan. I'll be back as soon as I can."`
```

	on complete
		event "deep: scan log research" 14
		log `Scanned Remnant ships beyond the wormhole in Terminus and handed the scan logs over to Ivan and his team.`
		dialog `The team is visibly excited when you return with the scanner logs. "Give us a few weeks to look these over," Ivan says. "Then we will contact you about what we want you to do next."`

conversation "deep: remnant c"
	`The team of scientists is already waiting in the spaceport when you arrive. They remove the equipment from your ship and immediately being analyzing the results after returning to the lab.`
	`	"At first glance, this series of wormholes seems no different than the one that is used to enter this region of space, but it will require some time for us to go over all the data the sensors collected," Ivan explains to you.`
	branch revealed
		has "deep: did reveal remnant"

> 	`	"While you wait, though, I have another job you could do. You mentioned something about strange alien creatures beyond the wormhole that seemed to be able to survive in the vacuum of space. Would you be able to return there and scan them for us? These alien creatures won't have 'outfits' for the scanners to pick up, but a typical outfits scanner is likely to give us the most information."`

Reviewer 3: ```suggestion
	`	"While you wait, though, I have another job you could do. You mentioned something about strange alien creatures beyond the wormhole that seemed to be able to survive in the vacuum of space. Would you be able to return there and scan them for us? These alien creatures won't have 'outfits' for the scanners to pick up, but a typical outfits scanner is still likely to give us the most information."`
```

mission "Deep: Wormhole Research"
	landing
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
		not system "Epsilon Leonis"
	destination "Valhalla"
	to offer
		has "event: deep: scan log research"
		has "deep: did not reveal remnant"
	on offer
		payment 250000
		dialog `You receive a message from Ivan after you land. "Thanks for all your help recently. We brought our findings to the Deep government and they gave us more funding! We've transferred <payment> to your account and made sure you were listed as a primary contributor on all the reports of these immense discoveries. We do not have any further jobs for you, but hopefully we will in the future. Thanks again!"`

> 			decline

Author: OK. Switched to a conversation. Thanks!
